tests/test_hvcs.py

```diff
@@ -445,7 +445,44 @@ def request_callback(request):
 
             with mock.patch.dict(os.environ, {"NETRC": netrc_file.name}):
                 status = Github.post_release_changelog(
-                    "relekang", "rmoq", "1.0.0", "text"
+                    "relekang", "rmoq", "1.0.0", "text", False
+                )
+                self.assertTrue(status)
+
+    @responses.activate
+    @mock.patch("semantic_release.hvcs.Github.token", return_value=None)
+    def test_should_create_prerelease_if_asked_for(self, mock_token):
```

Comment:
I think you've duplicated the test name with the other flavours of git remotes - have left a comment below.
I don't think copy-paste is needed compared to the existing test cases - it should be fine to just add
```python
@pytest.mark.parametrize("prerelease", [True, False])
```
to the existing test along with the `prerelease` parameter, if that's easier (helps avoid repetition)

edit: forgot to click "save" when submitting the review